src/components/SearchBarView.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { SafeAreaView, View, StyleSheet} from "react-native"
+import { Icon, Input } from 'react-native-elements';
+import { gql, useLazyQuery} from '@apollo/client'
+import * as GetTypes from '../typings/src/ts/types'
+
+// GraphQL fragment
+const COMPANY_TILE_DATA = gql` {
```

Comment:
This query needs to be redefined using a new query (to be added to the GraphQL schema) on the businesses type. This should look similar to the example defined [here](https://www.apollographql.com/docs/react/data/fragments/#example-usage).

Reply:
We have decided that this can wait for full integration from team backend once that is sorted.